.github/workflows/dapr-longhaul-nightly.yml

```diff
@@ -168,7 +167,7 @@ jobs:
           kubectl apply -f ./longhaul-test/azure-service-bus-pubsub.yml -n ${{ env.APP_NAMESPACE }}
           kubectl apply -f ./longhaul-test/binding.yaml -n ${{ env.APP_NAMESPACE }}
           kubectl apply -f ./longhaul-test/redis-pubsub.yaml -n ${{ env.APP_NAMESPACE }}
-          kubectl apply -f ./longhaul-test/redis-statestore.yaml -n ${{ env.APP_NAMESPACE }}
+          kubectl apply -f ./longhaul-test/blob-statestore.yaml -n ${{ env.APP_NAMESPACE }}
```

Comment:
To match the other azure components, can you add `azure-` as a prefix to this and make `binding.yaml` match as well?